cmd/verify/README.md

```diff
@@ -0,0 +1,169 @@
+# Device Firmware Verification Tool
+
+The `verify` command is a tool which will check that the firmware currently installed
+on an ArmoredWitness device is authentic and discoverable via firmware transparency.
+
+There are 3 types of firmware installed on an ArmoredWitness device:
+
+1. Bootloader
+2. Secure Monitor ("OS")
+3. Witness applet
+
+Each of these is stored alongside a `ProofBundle` which contains firmware transparency
+information about the firmware image itself.
+
+Broadly, the tool performs the following operations:
+
+1. Puts the device into recovery mode, where its MMC storage becomes visible as a USB Mass Storage device.
+2. Extracts the installed firmware & proof bundles for each of the 3 types of firmware
+(bootloader, OS, applet).
+3. For each of the firmware types, it:
+    * Verifies the signature on the associated `manifest`.
+    * Hashes the firmware image, and asserts that it matches the one in the `manifest`.
+    * Verifies that the firmware `manifest` is present within the expected firmware transparency log.
+
+For more detailed information about the firmware transparency concepts and metadata, please
+see the [firmware transparency](/docs/transparency.md) doc.
+
+## Usage
+
+This tool is only useful if you have a provisioned ArmoredWitness device to hand.
+
+Ensure that the device is completely unplugged, with neither USB nor LAN connected.
+
+Locate the small black switch on the underside of the board, this switch has two positions:
```

Comment:
This will need instructions for popping open the enclosure.

Reply:
I believe the enclosure (in theory) has a little aperture over the switch, so we should be ok